packages/components/molecules/f-skeleton-loader/src/components/skeletons/CarouselCardRestaurantData.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+    <div class="c-skeleton-loader-restaurantData c-skeleton-loader-bone">
+        <sentence />
+        <rating />
+    </div>
+</template>
+
+<script>
+import Sentence from './Sentence.vue';
+import Rating from './Rating.vue';
+
+export default {
+    components: { Sentence, Rating }
+};
+</script>
+
+<style lang="scss" scoped>
+.c-skeleton-loader-restaurantData {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    width: 96%;
+    margin: auto;
+    padding: spacing();
+
+    .c-skeleton-loader-sentence {
+        width: 25%;
+    }
+    /deep/ .c-skeleton-loader-sentence .c-skeleton-loader-text {
```

Comment:
same question as before: if we remove `.c-skeleton-loader-sentence` will it break anything? Less nesting - the better.

Reply:
and it might be more readable and maintanable to use scss nesting here.
```
/deep/ .c-skeleton-loader-text {
    width: 70%;
    height: spacing(x1.8);

    &:nth-child(2) {
        margin-bottom: 0;
    }
}
```

Reply:
Good spot thanks